CMakeLists.txt

```diff
@@ -158,6 +170,7 @@ install (FILES
     include/bitbuffer.hpp
     include/hamiltonian.hpp
     include/statevector.hpp
+	include/pinvoke_api.hpp
```

Comment:
Whitespace

Reply:
(I haven't read the full PR yet) shouldn't all of the changes here be gated behind an "ENABLE_CSHARP" feature flag?